ccpp/data/GFS_typedefs.F90

```diff
@@ -2371,7 +2377,7 @@ subroutine sfcprop_create (Sfcprop, Model)
     allocate (Sfcprop%hprime   (IM,Model%nmtvr))
     allocate (Sfcprop%dust12m_in  (IM,12,5))
     allocate (Sfcprop%smoke_RRFS(IM,24,2))
-    allocate (Sfcprop%smoke2d_RRFS(IM,4))
+    allocate (Sfcprop%smoke2d_RRFS(IM,5))
     allocate (Sfcprop%emi_in   (IM,1))
     allocate(Sfcprop%albdirvis_lnd (IM))
```

Comment:
spacing inconcistent here between `allocate` and `(`

Reply:
I made all allocate statements consistent in both GFS_ and CCPP_typedefs.F90 (e.g. `allocate (...)`)

Reply:
Much appreciated!